src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial.mdx

```diff
@@ -0,0 +1,480 @@
+```sql 
+SELECT average(duration) 
+FROM Transaction 
+SINCE 1 day ago
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 day ago query"
+  alt="A screenshot of a query selecting the average duration since 1 day ago"
+  src={queriesnrqltutorial9}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    We can also specify time ranges relative to when we make our queries by using keywords like `day`, `week`, `hour`, `minute`, or their plural equivalents. We can also use logical expressions like `SINCE` today or `SINCE` this week.
+```sql
+SELECT average(duration) 
+FROM Transaction 
+SINCE this week
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 week ago"
+  alt="A screenshot of a query selecting the average duration since 1 week ago"
+  src={queriesnrqltutorial10}
+/>
+  </Side>
+</SideBySide>
+<SideBySide>
+  <Side>
+    Here is another query that demonstrate relative time ranges. Note that when specifying a relative time, you need to include the keyword `ago`.
+```sql
+SELECT average(duration) 
+FROM Transaction 
+SINCE 1 week ago 
+UNTIL 2 days ago
+```
+  </Side>
+  <Side>
+    <img
+  title="Select the average duration since 1 week ago until 2 days ago"
+  alt="A screenshot of a query selecting the average duration since 1 week ago until 2 days ago"
+  src={queriesnrqltutorial11}
+/>
+  </Side>
+</SideBySide>
+Now we're getting even more control of our data. By adding a time frame to a query, we're define exactly which window of data we want to see. These controls let us zero in on the data we want by defining where the data reporting should start and end. In our examples, we saw that we can easily choose the window and use common terms like `week`, `day`, `hour`, `minute`, and their plurals. You can even use `month` and `year` if you're storing that much data in New Relic. Let's go to the next step: querying a time series.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="time-series"
+    title="4. Query time series"
+  >
+<SideBySide>
+  <Side>
+    A common NRQL use case is querying how values change over time. This type of query provides data for line charts, area charts, and other visualizations. These "Timeseries" queries in NRDB are easy to do! We've already run some queries to explore aggregation functions `average()`, `max()`, `min()`, `sum()`, and `count()`. Now, we can take any query made with aggregation functions and add the `TIMESERIES` keyword to plot the values over time.
```

Comment:
@jeff-colucci I think you could just remove the quotation marks around timeseries and make it lower case when you're not invoking the keyword. For example, you could probably say, "These timeseries queries...."

Reply:
Thank you!